bin/dry-run.rb

```diff
@@ -209,7 +209,17 @@
                   "available options depend on PACKAGE_MANAGER"
   opts.on("--updater-options OPTIONS", opts_opt_desc) do |value|
     $options[:updater_options] = value.split(",").map do |o|
-      [o.strip.downcase.to_sym, true]
+      if o.include?("=")
+        o.split("=", 2).map.with_index do |v, i|
+          if i == 0
+            v.strip.downcase.to_sym
```

Comment:
I'm able to reason about this given the context I'm currently working with (reviewing a PR with a description in front of me), but I wonder what the experience will be like for future me.

Is there a way we could provide more context inline (e.g. `o.include?("goprivate=")`)? Or, if this is more generic and meant for parsing several arguments, could we extract this logic into a named method? I'd be hesitant to add yet more methods to this script, but I think the trade-off would be worth it. What do you think?